onyo/lib/onyo.py

```diff
@@ -339,7 +307,7 @@ def _get_dirs(self) -> set[Path]:
         (except under .git).
         """
         log.debug('Acquiring list of directories')
-        dirs = {self.sanitize_path(x) for x in Path(self.root).glob('**/')
+        dirs = {Path(x).resolve() for x in Path(self.root).glob('**/')
```

Comment:
This seems wrong. The paths reported by `Path(self.root).glob('**/')` are relative to root, not to CWD. So, `resolve()` is not what needs to doing here.


```suggestion
        dirs = {self.root / x for x in Path(self.root).glob('**/')
```